src/modules/location/components/map/map.tsx

```diff
@@ -1,61 +1,87 @@
-import { type FC, useState, useEffect, useRef } from 'react';
+import {
+  type FC,
+  useState,
+  useEffect,
+  useRef,
+  useCallback,
+  useLayoutEffect,
+} from 'react';
 import type L from 'leaflet';
 import { MapContainer, TileLayer, Marker } from 'react-leaflet';
-
-import { ZoomControl } from '@/modules/location/components/zoom/zoom-control';
+// components
+import { ZoomControl } from '@/modules/location/components';
+// constants
 import {
   proUserMarkerIcon,
   viewerUserMarkerIcon,
 } from '@/modules/location/constants/map.constants';
+// hooks
+import { useCombinedRefs } from '@/modules/core/hooks/use-combined-refs';
+// styles
+import 'leaflet/dist/leaflet.css';
 
-import styles from './map.module.scss';
 import type { MapProps } from './map.interface';
+import styles from './map.module.scss';
 
-export const Map: FC<MapProps> = ({ markers }) => {
```

Comment:
I don't see map on the cromatic https://64a8455bc3e8e7a6fd23ea11-zdclqwmijs.chromatic.com/?path=/docs/location-ui-map--docs.

Reply:
fixed